### PR TITLE
feat: Add task to query Astra DB

### PR DESCRIPTION
**Pull request by loicmathieu**

### CODE REVIEW

File: src/main/java/io/kestra/plugin/astradb/AstraDbSession.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        @Schema(
+            title = "The hostname of the Astra DB server"
+        )
+        @PluginProperty(dynamic = true)
+        @NotNull
+        @NotEmpty
+        private String hostname;
+
+        @Schema(
+            title = "The port of th Astra DB server"
```
Comment (brian-mulier-p) [inline src/main/java/io/kestra/plugin/astradb/AstraDbSession.java:84]:
```suggestion
            title = "The port of the Astra DB server"
```

File: src/main/java/io/kestra/plugin/astradb/Query.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    )
+    @PluginProperty
+    @NotNull
+    protected AstraDbSession session;
+
+    @Override
+    protected CqlSession cqlSession(RunContext runContext) throws IllegalVariableEvaluationException {
+        return this.session.connect(runContext);
+    }
+
```
Comment (brian-mulier-p) [inline src/main/java/io/kestra/plugin/astradb/Query.java:54]:
```suggestion
```

File: src/main/java/io/kestra/plugin/astradb/Query.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+            title = "Send a CQL query to an Astra DB database",
+            code = {
+                "session:",
+                "  secureBundle: /path/to/secureBundle.zip",
+                "  keyspace: astradb_keyspace",
+                "  clientId: astradb_clientId",
+                "  clientSecret: astradb_clientSecret",
+                "cql: SELECT * FROM CQL_TABLE",
+            }
+        ),
```
Comment (brian-mulier-p) [inline src/main/java/io/kestra/plugin/astradb/Query.java:39]: Add an example for proxy

File: src/main/java/io/kestra/plugin/cassandra/AbstractQuery.java [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+@NoArgsConstructor
+public abstract class AbstractQuery extends Task implements RunnableTask<AbstractQuery.Output>, QueryInterface {
+    @Builder.Default
+    private boolean fetch = false;
+
+    @Builder.Default
+    private boolean store = false;
+
+    @Builder.Default
+    private boolean fetchOne = false;
```
Comment (brian-mulier-p) [inline src/main/java/io/kestra/plugin/cassandra/AbstractQuery.java:57]: Would be better to use the [storeType](https://kestra.io/plugins/plugin-neo4j/tasks/io.kestra.plugin.neo4j.query#storetype)

File: src/main/java/io/kestra/plugin/cassandra/CassandraDbSession.java [hunk 5]
```diff
@@ -105,24 +105,24 @@ CqlSession connect(RunContext runContext) throws IllegalVariableEvaluationExcept
...
+            title = "The hostname of the Cassandra server"
         )
         @PluginProperty(dynamic = true)
         @NotNull
         @NotEmpty
         String hostname;
 
         @Schema(
-            title = "The port of cassandra server"
+            title = "The port of th Cassandra server"
```
Comment (brian-mulier-p) [inline src/main/java/io/kestra/plugin/cassandra/CassandraDbSession.java:116]:
```suggestion
            title = "The port of the Cassandra server"
```

**Review by brian-mulier-p — APPROVED** [review]
Some slight changes but almost LGTM

File: src/main/java/io/kestra/plugin/cassandra/AbstractQuery.java (hunk 4 above)
Comment (loicmathieu) [reply src/main/java/io/kestra/plugin/cassandra/AbstractQuery.java:57]: It's a refactory of the Cassandra Query so it cannot be done without breaking the backward compatibility

File: src/main/java/io/kestra/plugin/astradb/Query.java (hunk 3 above)
Comment (loicmathieu) [reply src/main/java/io/kestra/plugin/astradb/Query.java:39]: I have no idea how to test proxy so I prefer not to add an example that I cannot validate.

**Pull request merged**